src/commands.py

```diff
@@ -0,0 +1,40 @@
+import click
+import subprocess
+import os
+
+@click.group()
+def protecc():
+    pass
+
+@protecc.command()
+def hello():
+    """Example script."""
+    click.echo('Hello World!')
```

Comment:
I think we should omit this

Reply:
Yup. I'll remove it with the next commit.